Makefile

```diff
@@ -99,6 +99,9 @@ SHA512SUM=$(shell command -v sha512sum || echo "shasum -a 512")
 # to update minikubes default, update deploy/addons/gvisor
 GVISOR_TAG ?= latest
 
+# auto-pause-hook tag to push changes to
+AUTOPAUSE_HOOK_TAG ?= 1.13
```

Comment:
lets not use the Dot notation here, it should be a simple One letter version,
like V1